updated the build image section, removed trademarks, minor edits (#53)
logdna/logdna-agent-v2: docs/README.md

>   * [Running as Non-Root](#running-as-non-root)

Reviewer: We should take this out for 2.2 GA Docs.

Reviewer 2: They'll disappear when I merge back to the 2.2 branch

## Building Agent v2

Obtain the source code to build the agent from our [GitHub repository](https://github.com/logdna/logdna-agent-v2). You can either download the source files as a .zip file, or clone the repository.

* Use the following commands to clone and then cd into this repository:

```
git clone https://github.com/logdna/logdna-agent-v2.git
cd logdna-agent-v2
```

> Next, select how you want to build the image: using Docker or an agent binary on Linux.

Reviewer 2: ```suggestion
Next, select how you want to build the agent: as a docker image or as a Linux binary.
```

logdna-agent-v2         dcd54a0             e471b3d8a409        22 seconds ago      135MB
```

> You can also obtain the image and review our tagging scheme on [DockerHub](https://hub.docker.com/r/logdna/logdna-agent.)

Reviewer 2: ```suggestion
You can also obtain the image and review our tagging scheme on [DockerHub](https://hub.docker.com/r/logdna/logdna-agent).
```
